requirements.txt

```diff
@@ -9,8 +11,10 @@ lxml
 mock
 pandas
 plotly
-pylint==2.4.4
```

Comment:
how confident are we that removing specific version is not causing issues somewhere else... a sanity question

Reply:
I've found that keeping linters/formatters to the latest version rarely (but likely not never) causes issues, and if it does it will cause some builds to fail because of new rules (or bug fixes that didn't apply previous rules), but no runtime errors will pop up randomly because of them

Overall this seems OK to me

Reply:
thanks for 2nd opinion and ok